docs:#799tiered storage support for azure (#1057)
redpanda-data/documentation: modules/get-started/pages/licensing/overview.adoc

[TIP]

> ====

Reviewer: ```suggestion
[TIP]
====
Go to the Redpanda Self-Managed quickstart to set up a local Redpanda cluster and try the Enterprise Edition free for up to 60 days.
====
```

[TIP]
====

> Go to the Redpanda Self-Managed quickstart to set up a local Redpanda cluster and try the Enterprise Edition free for up to 30 days.

Reviewer 2: I think 'up to 60 days' is still correct because you get 30 days by default and it's also free to extend for another 30 days.

> Go to the Redpanda Self-Managed quickstart to set up a local Redpanda cluster and try the Enterprise Edition free for up to 30 days.

Reviewer: ```suggestion
```

====
Go to the Redpanda Self-Managed quickstart to set up a local Redpanda cluster and try the Enterprise Edition free for up to 60 days.

> ====

Reviewer: ```suggestion
```

You can also generate a <<Trial license>> for the Enterprise Edition.

=== Trial license

All new Redpanda clusters automatically receive a trial license that's valid for 30 days. If you wish to extend this trial an additional 30 days, use the xref:reference:rpk/rpk-generate/rpk-generate-license.adoc[`rpk generate license`] command. This extended evaluation period begins after you apply the trial license to your cluster using the `--apply` flag. After this period expires, inactive enterprise features are disabled, and active features enter a restricted state.

IMPORTANT: Only one trial license is permitted per email and business domain. Attempts to generate multiple trial licenses from the same email or business domain result in a license error. To get a permanent license, contact https://www.redpanda.com/contact[Redpanda Sales^].

To generate and apply a 30-day trial license for Enterprise Edition:

[source,bash]
----

rpk generate license --apply
First Name: <your_first_name>
Last Name: <your_last_name>
Company name: <your company name>
Business Email: <your business email address>

> 
> Successfully saved license to "/home/<your local directory>/code/rp/redpanda/src/go/rpk/redpanda.license".
> 
> Upload this license in Redpanda Console, or run:
>   rpk cluster license set --path /home/<your local directory>/rp/redpanda/src/go/rpk/redpanda.license
> 
> This license expires on 2025-05-07.
> 
> For more information, see:
>   https://docs.redpanda.com/current/get-started/licensing/overview/#license-keys

Reviewer: ````suggestion
----

Example output:

```bash
Successfully saved license to "/home/<your local directory>/code/rp/redpanda/src/go/rpk/redpanda.license".

Upload this license in Redpanda Console, or run:
  rpk cluster license set --path /home/<your local directory>/rp/redpanda/src/go/rpk/redpanda.license

This license expires on 2025-05-07.

For more information, see:
  https://docs.redpanda.com/current/get-started/licensing/overview/#license-keys
```
````

== Enterprise Edition

The Enterprise Edition is licensed with the https://github.com/redpanda-data/redpanda/blob/dev/licenses/rcl.md[Redpanda Community License^] (RCL). It includes the free features licensed under the Redpanda BSL, as well as enterprise features.
You can also generate a <<Trial license>> for the Enterprise Edition.

=== Trial license

> All new Redpanda clusters automatically receive a trial license that's valid for 30 days. If you wish to extend this trial an additional 30 days, use the xref:reference:rpk/rpk-generate/rpk-generate-license.adoc[`rpk generate license`] command. This extended evaluation period begins after you apply the trial license to your cluster using the `--apply` flag. After this period expires, inactive enterprise features are disabled, and active features enter a restricted state.

Reviewer: ```suggestion
```

> 

Reviewer: ```suggestion
All new Redpanda clusters automatically receive a trial license valid for 30 days. To extend this trial for 30 days, use the xref:reference:rpk/rpk-generate/rpk-generate-license.adoc[`rpk generate license`] command or request one at [https://redpanda.com/try-enterprise[redpanda.com^]](https://redpanda.com/try-enterprise%5Bredpanda.com%5E%5D). This extended evaluation period begins after you apply the trial license to your cluster using the `--apply` flag. After this period expires, inactive enterprise features are disabled, and active features enter a restricted state.
```

IMPORTANT: Only one trial license is permitted per email and business domain. Attempts to generate multiple trial licenses from the same email or business domain will result in a license error. To get a permanent license, contact https://www.redpanda.com/contact[Redpanda Sales^].

To generate and apply a 30-day trial license for Enterprise Edition:

[source,bash,subs="attributes+"]
----

rpk generate license --apply
First Name: <your_first_name>
Last Name: <your_last_name>
Company name: <your company name>
Email: <your business email address>

Successfully saved license to "/home/<your local directory>/code/rp/redpanda/src/go/rpk/redpanda.license".

Upload this license in Redpanda Console, or run:
  rpk cluster license set --path /home/<your local directory>/rp/redpanda/src/go/rpk/redpanda.license

This license expires on 2025-05-07.

For more information, see:
  https://docs.redpanda.com/current/get-started/licensing/overview/#license-keys
----

> 

Reviewer: ```suggestion

Optionally, you can https://redpanda.com/try-enterprise[request a new trial license key] to extend your trial period.
```

IMPORTANT: Only one trial license is permitted per email and business domain. Attempts to generate multiple trial licenses from the same email or business domain will result in a license error. To get a permanent license, contact https://www.redpanda.com/contact[Redpanda Sales^].

To generate and apply a 30-day trial license for Enterprise Edition:

[source,bash,subs="attributes+"]
----

rpk generate license --apply
First Name: <your_first_name>
Last Name: <your_last_name>
Company name: <your company name>
Email: <your business email address>

Successfully saved license to "/home/<your local directory>/code/rp/redpanda/src/go/rpk/redpanda.license".

Upload this license in Redpanda Console, or run:
  rpk cluster license set --path /home/<your local directory>/rp/redpanda/src/go/rpk/redpanda.license

This license expires on 2025-05-07.

For more information, see:
  https://docs.redpanda.com/current/get-started/licensing/overview/#license-keys
----

> Optionally, you can https://redpanda.com/try-enterprise[request a new trial license key] to extend your trial period.

Reviewer: ```suggestion
Optionally, you can https://redpanda.com/try-enterprise[request a new trial license key] to extend your trial period.

```

IMPORTANT: Only one trial license is permitted per email and business domain. Attempts to generate multiple trial licenses from the same email or business domain will result in a license error. To get a permanent license, contact https://www.redpanda.com/contact[Redpanda Sales^].

To generate and apply a 30-day trial license for Enterprise Edition:

[source,bash,subs="attributes+"]
----

rpk generate license --apply
First Name: <your_first_name>
Last Name: <your_last_name>
Company name: <your company name>
Email: <your business email address>

Successfully saved license to "/home/<your local directory>/code/rp/redpanda/src/go/rpk/redpanda.license".

Upload this license in Redpanda Console, or run:
  rpk cluster license set --path /home/<your local directory>/rp/redpanda/src/go/rpk/redpanda.license

This license expires on 2025-05-07.

For more information, see:
  https://docs.redpanda.com/current/get-started/licensing/overview/#license-keys
----

> Optionally, you can https://redpanda.com/try-enterprise[request a new trial license key] to extend your trial period.

Reviewer: ```suggestion
NOTE: To activate the trial license, you must apply it to your cluster using the `--apply` flag. The trial license is saved in your working directory or the specified path, based on the `--path` flag.

Optionally, you can https://redpanda.com/try-enterprise[request a new trial license key] to extend your trial period.
```

=== License keys

Enterprise features require an Enterprise Edition license key, sometimes called enterprise license, license key, or license.

- **Redpanda**: Starting with version 24.3, new Redpanda clusters automatically receive a trial license that's valid for 30 days, allowing unrestricted use of enterprise features. This evaluation period begins when the cluster is created for the first time. After this period expires, inactive enterprise features are disabled, and active features enter a restricted state.
+

> To extend your trial license key see <<Trial license>>, or https://redpanda.com/upgrade[upgrade to Redpanda Enterprise^].

Reviewer: ```suggestion
To extend your trial license ` see <<Trial license>>, https://redpanda.com/try-enterprise[request a new trial license key], or https://redpanda.com/upgrade[upgrade to Redpanda Enterprise^].
```

== Enterprise Edition

The Enterprise Edition is licensed with the https://github.com/redpanda-data/redpanda/blob/dev/licenses/rcl.md[Redpanda Community License^] (RCL). It includes the free features licensed under the Redpanda BSL, as well as enterprise features.
You can also generate a <<Trial license>> for the Enterprise Edition.

=== Trial license

All new Redpanda clusters automatically receive a trial license that's valid for 30 days. If you wish to extend this trial an additional 30 days, use the xref:reference:rpk/rpk-generate/rpk-generate-license.adoc[`rpk generate license`] command. This extended evaluation period begins after you apply the trial license to your cluster using the `--apply` flag. After this period expires, inactive enterprise features are disabled, and active features enter a restricted state.

IMPORTANT: Only one trial license is permitted per email and business domain. Attempts to generate multiple trial licenses from the same email or business domain result in a license error. To get a permanent license, contact https://www.redpanda.com/contact[Redpanda Sales^].

To generate and apply a 30-day trial license for Enterprise Edition:

> [source,bash,subs="attributes+"]

Reviewer 2: ```suggestion
[source,bash]
```

----

rpk generate license --apply
First Name: <your_first_name>
Last Name: <your_last_name>
Company name: <your company name>

> Email: <your business email address>

Reviewer: ```suggestion
Business Email: <your business email address>
```

Successfully saved license to "/home/<your local directory>/code/rp/redpanda/src/go/rpk/redpanda.license".

Upload this license in Redpanda Console, or run:
  rpk cluster license set --path /home/<your local directory>/rp/redpanda/src/go/rpk/redpanda.license

This license expires on 2025-05-07.

For more information, see:

>   https://docs.redpanda.com/current/get-started/licensing/overview/#license-keys

Reviewer 2: Please could we move this out into a separate ' Example output' block? This will allow users to copy/paste the command.

----

NOTE: To activate the trial license, you must apply it to your cluster using the `--apply` flag. The trial license is saved in your working directory or the specified path, based on the `--path` flag.

> Optionally, you can https://redpanda.com/try-enterprise[request a new trial license key] to extend your trial period.

Reviewer 2: It's easy to miss this. I think it might be worth putting it closer to the top. Something like:

> All new Redpanda clusters automatically receive a trial license valid for 30 days. To extend this trial for 30 days, use the xref:reference:rpk/rpk-generate/rpk-generate-license.adoc[`rpk generate license`] command or request one at https://redpanda.com/try-enterprise[redpanda.com^]. This extended evaluation period begins after you apply the trial license to your cluster using the `--apply` flag. After this period expires, inactive enterprise features are disabled, and active features enter a restricted state.

Reviewer: Agree—thx

=== License keys

Enterprise features require an Enterprise Edition license key, sometimes called enterprise license, license key, or license.

- **Redpanda**: Starting with version 24.3, new Redpanda clusters automatically receive a trial license that's valid for 30 days, allowing unrestricted use of enterprise features. This evaluation period begins when the cluster is created for the first time. After this period expires, inactive enterprise features are disabled, and active features enter a restricted state.
+

> To extend your trial license ` see <<Trial license>>, https://redpanda.com/try-enterprise[request a new trial license key], or https://redpanda.com/upgrade[upgrade to Redpanda Enterprise^].

Reviewer: ```suggestion
To extend your trial license see <<Trial license>>, https://redpanda.com/try-enterprise[request a new trial license key], or https://redpanda.com/upgrade[upgrade to Redpanda Enterprise^].
```

== Enterprise Edition

The Enterprise Edition is licensed with the https://github.com/redpanda-data/redpanda/blob/dev/licenses/rcl.md[Redpanda Community License^] (RCL). It includes the free features licensed under the Redpanda BSL, as well as enterprise features.
You can also generate a <<Trial license>> for the Enterprise Edition.

=== Trial license

> All new Redpanda clusters automatically receive a trial license valid for 30 days. To extend this trial for 30 days, use the xref:reference:rpk/rpk-generate/rpk-generate-license.adoc[`rpk generate license`] command or request one at [https://redpanda.com/try-enterprise[redpanda.com^]](https://redpanda.com/try-enterprise%5Bredpanda.com%5E%5D). This extended evaluation period begins after you apply the trial license to your cluster using the `--apply` flag. After this period expires, inactive enterprise features are disabled, and active features enter a restricted state.
> IMPORTANT: Only one trial license is permitted per email and business domain. Attempts to generate multiple trial licenses from the same email or business domain result in a license error. To get a permanent license, contact https://www.redpanda.com/contact[Redpanda Sales^].

Reviewer: ```suggestion
All new Redpanda clusters automatically receive a trial license valid for 30 days. To extend this trial for 30 days, use the xref:reference:rpk/rpk-generate/rpk-generate-license.adoc[`rpk generate license`] command, or request one at https://redpanda.com/try-enterprise[redpanda.com^]. This extended evaluation period begins after you apply the trial license to your cluster using the `--apply` flag. After this period expires, inactive enterprise features are disabled, and active features enter a restricted state.

IMPORTANT: Only one trial license is permitted per email and business domain. Attempts to generate multiple trial licenses from the same email or business domain result in a license error. To get a permanent license, contact https://www.redpanda.com/contact[Redpanda Sales^].
```

IMPORTANT: Only one trial license is permitted per email and business domain. Attempts to generate multiple trial licenses from the same email or business domain result in a license error. To get a permanent license, contact https://www.redpanda.com/contact[Redpanda Sales^].

To generate and apply a 30-day trial license for Enterprise Edition:

[source,bash]
----

rpk generate license --apply
First Name: <your_first_name>
Last Name: <your_last_name>
Company name: <your company name>
Business Email: <your business email address>
----

> 
> Example output:

Reviewer: ```suggestion

NOTE: To activate the trial license, you _must_ apply it to your cluster using the `--apply` flag. The trial license is saved in your working directory or the specified path, based on the `--path` flag.

Optionally, you can https://redpanda.com/try-enterprise[request a new trial license key] to extend your trial period.

Example output:
```

== Enterprise Edition

The Enterprise Edition is licensed with the https://github.com/redpanda-data/redpanda/blob/dev/licenses/rcl.md[Redpanda Community License^] (RCL). It includes the free features licensed under the Redpanda BSL, as well as enterprise features.
You can also generate a <<Trial license>> for the Enterprise Edition.

=== Trial license

> All new Redpanda clusters automatically receive a trial license valid for 30 days. To extend this trial for 30 days, use the xref:reference:rpk/rpk-generate/rpk-generate-license.adoc[`rpk generate license`] command, or request one at https://redpanda.com/try-enterprise[redpanda.com^]. This extended evaluation period begins after you apply the trial license to your cluster using the `--apply` flag. After this period expires, inactive enterprise features are disabled, and active features enter a restricted state.

Reviewer: ```suggestion
All new Redpanda clusters automatically receive a trial license valid for 30 days. To extend this trial for 30 days, use the xref:reference:rpk/rpk-generate/rpk-generate-license.adoc[`rpk generate license`] command, or  https://redpanda.com/try-enterprise[request a new trial license key^]. This extended evaluation period begins after you apply the trial license to your cluster using the `--apply` flag. After this period expires, inactive enterprise features are disabled, and active features enter a restricted state.
```

IMPORTANT: Only one trial license is permitted per email and business domain. Attempts to generate multiple trial licenses from the same email or business domain result in a license error. To get a permanent license, contact https://www.redpanda.com/contact[Redpanda Sales^].

To generate and apply a 30-day trial license for Enterprise Edition:

[source,bash]
----

rpk generate license --apply
First Name: <your_first_name>
Last Name: <your_last_name>
Company name: <your company name>
Business Email: <your business email address>
----

NOTE: To activate the trial license, you _must_ apply it to your cluster using the `--apply` flag. The trial license is saved in your working directory or the specified path, based on the `--path` flag.

> Optionally, you can https://redpanda.com/try-enterprise[request a new trial license key] to extend your trial period.

Reviewer: ```suggestion
```

Successfully saved license to "/home/<your local directory>/code/rp/redpanda/src/go/rpk/redpanda.license".

Upload this license in Redpanda Console, or run:
  rpk cluster license set --path /home/<your local directory>/rp/redpanda/src/go/rpk/redpanda.license

This license expires on 2025-05-07.

For more information, see:
  https://docs.redpanda.com/current/get-started/licensing/overview/#license-keys
```

> ----

Reviewer: ```suggestion
```

To generate and apply a 30-day trial license for Enterprise Edition:

[source,bash]
----

rpk generate license --apply
First Name: <your_first_name>
Last Name: <your_last_name>
Company name: <your company name>
Business Email: <your business email address>
----

NOTE: To activate the trial license, you _must_ apply it to your cluster using the `--apply` flag. The trial license is saved in your working directory or the specified path, based on the `--path` flag.

Optionally, you can https://redpanda.com/try-enterprise[request a new trial license key] to extend your trial period.

Example output:

```bash
Successfully saved license to "/home/<your local directory>/code/rp/redpanda/src/go/rpk/redpanda.license".

Upload this license in Redpanda Console, or run:
  rpk cluster license set --path /home/<your local directory>/rp/redpanda/src/go/rpk/redpanda.license

This license expires on 2025-05-07.

For more information, see:
  https://docs.redpanda.com/current/get-started/licensing/overview/#license-keys
```

> NOTE: To activate the trial license, you must apply it to your cluster using the `--apply` flag. The trial license is saved in your working directory or the specified path, based on the `--path` flag.
> 
> Optionally, you can https://redpanda.com/try-enterprise[request a new trial license key] to extend your trial period.

Reviewer: ```suggestion
```
